plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -34,7 +34,7 @@ func calculateScore(requested int64, capacity int64, node string) int {
 		return 0
 	}
 	if requested > capacity {
-		glog.Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
+		glog.V(4).Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
```

Comment:
Works for me.  Could probably stand to have someone check out V level consistency across the codebase rather than case-by-case.  FWIW I believe you get all of the reasons a pod failed to fit on a given node in the other `V(2)` statement in this PR.